roles/postgresql/defaults/main.yml

```diff
@@ -70,6 +70,8 @@ postgresql_additional_settings: []
 
 postgresql_settings: "{{ postgresql_generated_settings + postgresql_additional_settings }}"
 
+jdbc_connector_package: postgresql-jdbc
```

Comment:
All variable should start with `postgresql_` like other variables.

Reply:
Done, called `postgresql_jdbc_connector_package` now.